example_helper_test.go

```diff
@@ -23,5 +23,5 @@ func ExampleHelper() {
 	ctx := context.Background()
 	httpLogHelper(ctx, http.StatusBadGateway)
 
-	// 2019-12-07 21:18:42.567 [INFO]	<example_helper_test.go:24>	sending HTTP response	{"status": 502}
+	// Dec  7 21:18:42.567 [INFO]	<example_helper_test.go:24>	sending HTTP response	{"status": 502}
```

Comment:
iirc the purpose with the full date/time is so that log files can be sorted correctly.

like if you merge the logs from machine a and machine b into a single file, you can just run `sort` to get the file sorted by timestamp.

Reply:
I do agree, but I would think `sort` wouldn't be able to handle the stack frames for `slog.Error` logs.

Reply:
Good point!

Reply:
Yea with sloghuman's multiline formatting, sorting isn't a real concern.

I'm not convinced this is a good move though. If I was going through old logs, I'd prefer to be able to just grep for something and then find the log I'm looking for and see their exact date rather than having to get the date from the file the log is stored in or something else auxillary. So I'd prefer to leave things as they are.